### ISSUE TITLE
update our listing on Ulife

### ISSUE THREAD

**Issue by SaraMati** [UofTCoders/council#305]
It seems that we are not listed currently on [ULife ](https://www.ulife.utoronto.ca/page/view/slug/renewgrouprecognition)(people can't find us as a UofT student group/club on Ulife's website). 
we have to either renew our recognition every year to appear under university of toronto students club or simply update our profile. 

If I understand correctly, at this point the last person who was primary contact on our profile (probably Lina, the previous president?) should contact groups.officer@utoronto.ca and let them know that the primary contact is passed on to the new president. otherwise, I'd be happy to take care of the correspondence with the office (CCing Ahmed). 

or if I'm completely wrong and our profile is update, please kindly point me to it.

**Comment by aays** [comment]
Hey @SaraMati - thanks for this, and apologies for completely missing it earlier somehow! I believe we did update our profile to make me the primary contact in 2019, but they send out an email mentioning the following:

> If you are renewing a group, we encourage you to submit your renewal abiding by the updated policy for the September 30, 2020 deadline. However, if you are unable to complete all required changes of the updated policy before this year's renewal deadline, our expectation is that you will make the required changes to your group so they will be in place for the next renewal deadline of September 30, 2021.

Since we were kind of in flux at the time and not sure about the state of Coders, I figured the best course of action was to pass that deadline and do the renewal process in time for Sept 2021, but I didn't realize that this would mean we'd be removed from the website! I think it's definitely worth emailing them to sort this out if you're still up for it, especially now that we were finally able to solidify our 2020-2021 exec in October 2020 or so. I think we'd have to be officially registered as 'University of Toronto Coders Student Group' based on their new policy but there isn't much else that changes I believe.
